### PR TITLE
Matomo P3: Update Footer Tracking

### PR DESCRIPTION
**Pull request by aburke07**
## Overview 
This pull updates the current tracking function we already use for the footer to match the spec from [the spreadsheet](https://docs.google.com/spreadsheets/d/16Xas1pHf_K7FmfOX3yX2mYgeYr2DIENr8cyQp19FZkA/edit?pli=1#gid=1839136207&fvid=1617862179).

We already had this tracking mechanism in place, so this just modifies what we actually send in the event action as well as adds tracking to the store change button.

## QA
Check that the event shows up in matomo or GA for events 125-143 from [the spreadsheet](https://docs.google.com/spreadsheets/d/16Xas1pHf_K7FmfOX3yX2mYgeYr2DIENr8cyQp19FZkA/edit?pli=1#gid=1839136207&fvid=1617862179).

Closes https://github.com/iFixit/ifixit/issues/44680 (and lines 125-143 from the spreadsheet)

### CODE REVIEW

**Comment by jordycosta** [comment]
QA 🟢 

matomo events 125-143 in [the spreadsheet](https://docs.google.com/spreadsheets/d/16Xas1pHf_K7FmfOX3yX2mYgeYr2DIENr8cyQp19FZkA/edit?pli=1#gid=1839136207&fvid=1617862179) are being tracked 👍 

**View [Visitor Profile](https://matomo.ubreakit.com/index.php?module=CoreHome&action=index&idSite=1&period=day&date=yesterday#?period=day&date=2022-11-28&idSite=1&category=General_Visitors&subcategory=Live_VisitorLog&popover=visitorProfile$3Aa798e9c1ef5d994b)**

File: packages/footer/hooks/useTrackedOnClick.tsx [hunk 1]
```diff
@@ -4,18 +4,25 @@ import { TrackingContext } from './TrackingContext';
...
+   isStoreLink?: boolean;
 };
 
 export function useTrackedOnClick<T>({
    href,
    onClick,
+   clickName,
+   isStoreLink,
 }: UseFooterLinkTrackingProps<T>) {
+   const LinkType = isStoreLink ? 'Stores' : 'iFixit';
```
Comment (jyee27) [inline packages/footer/hooks/useTrackedOnClick.tsx:17]: (minor) We should probably name this `linkType` since it's a regular variable.

**Review by jyee27 — APPROVED** [review]
CR ☕ not blocking on the comment

File: packages/footer/hooks/useTrackedOnClick.tsx (hunk 1 above)
Comment (aburke07) [reply packages/footer/hooks/useTrackedOnClick.tsx:17]: Good catch.

Comment (aburke07) [reply packages/footer/hooks/useTrackedOnClick.tsx]: Fixed in [1f47c68](https://github.com/iFixit/react-commerce/pull/1069/commits/1f47c689fb1fe0a001515ce91f1e717afe0927ee)

**Comment by aburke07** [comment]
QA ↪️  carryover - just renamed a variable

**Review by jyee27 — APPROVED** [review]
CR ☕

**Pull request merged**